samples/apps/logging/logging.cpp

```diff
@@ -756,6 +756,8 @@ namespace loggingapp
         .set_auto_schema<void, LoggingGetHistorical::Out>()
         .add_query_parameter<size_t>("id")
         .set_forwarding_required(ccf::endpoints::ForwardingRequired::Never)
+        .set_execute_outside_consensus(
```

Comment:
I am worried by this. We've historically only used this for node-specific data, which was clearly out of consensus. For business/application endpoints though, the expectation/reality so far is that consensus guarantees apply evenly.

We are lacking clear guidance of when to use this flag and what the implications are, I think this is something we need to address.